src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column.js

```diff
@@ -96,6 +121,43 @@ define(
              */
             getSessionStorageKey: function () {
                 return 'collapsedColumn_' + this.config.stateCode;
+            },
+
+            /**
+             * Registers a new item to display on navigation template
+             *
+             * @param {Event}    navigationItem
+             * @param {string}   navigationItem.label
+             * @param {function} navigationItem.isVisible
+             * @param {string}   navigationItem.code
+             */
+            registerNavigationItem: function (navigationItem) {
+                this.navigationItems.push(navigationItem);
+            },
+
+            /**
+             * Returns the visible navigation items
+             *
+             * @returns {Array}
+             */
+            getNavigationItems: function () {
+                return _.filter(this.navigationItems, function (navigationItem) {
+                    return !_.isFunction(navigationItem.isVisible) || navigationItem.isVisible();
+                });
+            },
+
+            /**
+             * @param {Event} event
+             */
+            redirect: function (event) {
```

Comment:
I will let it as it because it's overriden by another column with "redirect"... More standard IMHO.